### ISSUE TITLE
Pas une proposition de traduction mais...

### ISSUE THREAD

**Issue by maaaaz** [soulaklabs/bitoduc.fr#109]
...au même titre qu'un smartphone est un **"ordiphone"**, je propose que des lunettes à réalité augmentée (par ex Google Glass) constituent un **"kaléidordinateur"**.

Cordialement.

**Comment by progval** [comment]
Traduction pour « smart glasses », donc ?

**Comment by maaaaz** [comment]
Oui !

**Comment by progval** [comment]
À vrai dire, dans Bitoduc, nous avons « malinphone » pour « smartphone ».
Pour la cohérence, je propose donc « malignettes » (lunettes malignes)

**Comment by maaaaz** [comment]
Je préfère mon kaléidordinateur :)

**Comment by p0nce** [comment]
Il faut penser aux montres intelligentes.
Perso je suis plutôt pour:
- « téléphones intelligents »
- « lunettes intelligentes »
- « montres intelligentes »

Ce qui est quand même très intelligent.

**Comment by Aluriak** [comment]
L'académie française à déjà fournis des traductions, dont [ordiphone](https://huit.re/_YMMeoRH). Pourquoi ne pas encourager l'usage de ces traductions ?

**Comment by p0nce** [comment]
@Aluriak Une très bonne question dont la réponse est couverte par le fichier https://github.com/soulaklabs/bitoduc.fr/blob/gh-pages/CONTRIBUER.md

**Comment by Aluriak** [comment]
Effectivement, l'argument se tient. Je resterais personellement sur ordiphone, nous verrons bien lequel sera le plus usité. Pour le moment, [c'est équivalent](https://books.google.com/ngrams/graph?content=Ordiphone%2CMalinphone&case_insensitive=on&year_start=1500&year_end=2016&corpus=19&smoothing=1&share=&direct_url=) :)
Merci !

**Comment by maaaaz** [comment]
Je tenais beaucoup à mon "kaléidordinateur", "ordiphone" n'est pas assez spécifique aux lunettes connectées :(

**Comment by Aluriak** [comment]
Je trouve «kaléidordinateur» est un peu compliqué à lire et à prononcer. Pourquoi pas «kaléinateur» ? 
On perd peut-être le sens, sans le `d`… «kaléidinateur» est peut-être mieux ?

«Kalordinateur» se confond avec _cale l'ordinateur_ ou _car l'ordinateur_, donc pas top.

**Comment by chmduquesne** [comment]
Kaléidordinateur pose un problème d'éthymologie. D'après wiktionary.org, le
mot kaléidoscope est "dérivé du grec ancien καλός
https://fr.wiktionary.org/wiki/%CE%BA%CE%B1%CE%BB%CF%8C%CF%82 _kalós_
(« beau »), εἶδος
https://fr.wiktionary.org/wiki/%CE%B5%E1%BC%B6%CE%B4%CE%BF%CF%82 _eîdos_
(« forme ») et σκοπέω
https://fr.wiktionary.org/wiki/%CF%83%CE%BA%CE%BF%CF%80%CE%AD%CF%89
_skopéô_ (« observer »)". Votre composition signifierait donc "ordinateur à
belles formes", et non un ordinateur par lequel on peut observer. Il serait
de meilleur ton de créer un mot à partir de la racine "scope", qui traduit
mieux l'usage associé à l'objet.

2016-11-07 21:01 GMT+01:00 Lucas Bourneuf notifications@github.com:
